### ISSUE TITLE
Adding a non-string to a list

### ISSUE THREAD

**Issue by WillerZ** [bazelbuild/buildtools#565]
It seems that [the logic for the `add` command](https://github.com/bazelbuild/buildtools/blob/34d7371037136c478031f03a11692fb16ec5e11a/edit/buildozer.go#L81-L92) to buildozer adds `"` quotes around the item to be added unless it knows that the list it's adding to is a list of ints.

Can you add an `add_raw` command for the (admittedly rare) scenario that someone wants to add a non-string item to a list. For my use-case all of the `add`s I want to do are of this nature, so a commandline flag to change the behavior of `add` for the duration of one invocation would also work for me.

**Comment by adzenith** [comment]
I am running into this issue right now. I'm trying to add `requirement("package_name")` to a `deps` list for use with [rules_python](https://github.com/bazelbuild/rules_python), but it ends up like:
```
$ buildozer 'add deps foo("bar")' //:target
fixed BUILD
$ buildozer 'print deps' //:target
[
    "foo(\"bar\")",
]

**Comment by vladmos** [comment]
The problem is that buildozer knows that `deps` contains a list of strings and converts any input regardless of whether it was quoted or not to a string literal.

If I change that to "don't use string literals unless the input is quoted" that will be a breaking change, many scripts that use buildozer will be broken.

What do you think about the following syntax:

    buildozer 'add deps `foo("bar")`' //:target

to emphasize that the argument should be inserted as is?

**Comment by adzenith** [comment]
Yeah, that could work - or the `add_raw` command suggested in the initial comment. Right now I’m working around the issue by adding aliases for the requirements that I’m using so that the aliases are just strings.

**Comment by WillerZ** [comment]
We are using `add` with a reasonably unique suffix and then `sed` to fix the results.

**Comment by wyattanderson** [comment]
This would be a really positive change for me and would eliminate a lot of hacky workarounds. Either the `_raw` variants or the backticks would be fine; I think a command-line flag that was applied to all options would work as well—I could modify my tools to add quotes where necessary.

**Comment by adzenith** [comment]
I would still love to be able to do this
